pkg/scheduler/scheduler.go

```diff
@@ -61,6 +62,12 @@ const (
 	// ringCheckPeriod is how often we check the ring to see if this instance is still in
 	// the replicaset of instances to act as schedulers.
 	ringCheckPeriod = 3 * time.Second
+
+	// ActorPathHeader is the name of the header used to enqueue requests in hierarchical queues
+	ActorPathHeader = "x-actor-path"
```

Comment:
I wonder, whether we should start using `X-Loki-*` headers consistently, like here for the query limits: https://github.com/grafana/loki/pull/8727/files#diff-a19fa86227cdb8ba8ac4d877d8f8fa4337c00ab92594fae43cdae5b9b5c10a96R12

```suggestion
	ActorPathHeader = "x-loki-actor-path"
```